grails-app/controllers/org/pih/warehouse/inventory/StockMovementController.groovy

```diff
@@ -313,6 +318,14 @@ class StockMovementController {
         [stockMovement: stockMovement, comment: new Comment()]
     }
 
+    def rejectRequestWithComment = {
```

Comment:
Do you have to be that explicit here? I think `reject` or `rejectRequest` would be fine if the comment is mandatory.

Reply:
Yes please rename as reject.